src/CalculatorUnitTests/DateCalculatorUnitTests.cpp

```diff
@@ -387,6 +387,63 @@ namespace DateCalculationUnitTests
             VERIFY_IS_TRUE(StringReference(L"") != viewModel->StrDateResult);
         }
 
+        TEST_METHOD(DateCalcViewModelDateDiffDaylightSavingTimeTest)
+        {
+          auto viewModel = ref new DateCalculatorViewModel();
+
+          viewModel->IsDateDiffMode = true;
+          VERIFY_IS_TRUE(viewModel->IsDateDiffMode);
+
+          // 29.02.2008
```

Comment:
It looks like you forgot to make these two dates  be M-D-Y like the others that you had changed in a previous update.

Reply:
all the dates in this file are dd.mm.yyyy (see line 275). The few dates I added use the same format

Reply:
Right you are.  I misread the code.  Apologies.